[SYNTH-19785] Add `selectiveRerun` input (#225)
DataDog/datadog-ci-azure-devops: README.md

| `apiKey`               | Your Datadog API key. This key is [created in your Datadog organization][6] and should be stored as a [secret][7]. <br><sub>**Required** when `authenticationType == apiAppKeys`</sub>                                                                                                                                  |
| `appKey`               | Your Datadog application key. This key is [created in your Datadog organization][6] and should be stored as a [secret][7]. <br><sub>**Required** when `authenticationType == apiAppKeys`</sub>                                                                                                                          |
| `authenticationType`   | (**Required**) How to store and retrieve credentials. <br><sub>Must be either `apiAppKeys` or `connectedService`</sub>                                                                                                                                                                                                  |

> | `batchTimeout`         | The duration in milliseconds after which the CI batch fails as timed out. This does not affect the outcome of a test run that already started. <br><sub>**Default:** `1800000` (30 minutes)</sub>                                                                                                                       |

Reviewer: What is the duration? The wait time?

Reviewer: How about this?
```suggestion
| `batchTimeout`         | Specifies the timeout duration (in milliseconds) for the CI batch. This does not affect the outcome of a test run that already started. <br><sub>**Default:** `1800000` (30 minutes)</sub>                                                                                                                       |
```

Author: @maycmlee The duration is the maximum duration of a batch. I like your suggestion 😄 

We can improve the second sentence too:

```md
Specifies the timeout duration in milliseconds for the CI batch. When a batch times out, the CI job fails, no new test runs are triggered, and ongoing test runs complete normally.
```

What do you think?

Reviewer: Hi @Drarig29! A couple of small suggestions for your revised second sentence, how about this?

> When a batch times out, the CI job fails and no new test runs are triggered, but ongoing test runs complete normally.

Author: Thanks! I'll change the `batchTimeout` and `selectiveRerun` descriptions across all repos

| `files`                | Glob patterns to detect Synthetic [test configuration files][14]. <br><sub>**Default:** `{,!(node_modules)/**/}*.synthetics.json`</sub>                                                                                                                                                                                 |
| `jUnitReport`          | The filename for a JUnit report if you want to generate one. <br><sub>**Default:** none</sub>                                                                                                                                                                                                                           |
| `publicIds`            | Public IDs of Synthetic tests to run, separated by new lines or commas. If no value is provided, tests are discovered in Synthetic [test configuration files][14]. <br><sub>**Default:** none</sub>                                                                                                                     |

> | `selectiveRerun`       | Whether to only rerun failed tests. If a test has already passed for a given commit, it will not be rerun in subsequent CI batches. By default, your [organization's default setting][18] is used. Set it to `false` to force full runs when your configuration enables it by default. <br><sub>**Default:** none</sub> |

Reviewer: ```suggestion
| `selectiveRerun`       | Whether to only rerun failed tests. If a test has already passed for a given commit, it is not rerun in subsequent CI batches. By default, your [organization's default setting][18] is used. Set it to `false` to force full runs when your configuration enables it by default. <br><sub>**Default:** none</sub> |
```
